contracts/src/v0.8/functions/dev/1_0_0/FunctionsRouter.sol

```diff
@@ -244,6 +245,11 @@ contract FunctionsRouter is IFunctionsRouter, FunctionsSubscriptions, Pausable,
       })
     );
 
+    // Do not allow setting a comittment for a requestId that already exists
```

Comment:
Is this just a sanity check in case, by some incredibly unlucky chance, we encounter a collision?

Reply:
That or a malicious Coordinator passes an intentional duplicate. Just something that should never happen.